brainstormforce/ultimate-addons-for-gutenberg
# fix: Improved the Logic & Made some settings Changes.

### Description
fix: Improved the Logic & Made some settings Changes.

### Screenshots
Issue Screencast - https://share.bsf.io/RBuyXbbQ
Issue Fixed Screencast - https://share.bsf.io/9ZuDN6O1

### Types of changes
Bug fix (non-breaking change which fixes an issue) 

### How has this been tested?
 - See the above 2 videos for better understanding.
 - Check we get the Copy Paste Settings if we multi select blocks.
 - To select multiple blocks, just 1st select one block then by pressing ctrl + shift click on the block till which you want to select.
 - ALso check the settings is showing in desired blocks only. ie Only UAG & Core, Not displyaed in Pricelist child, FAQ Child, Google Maps, Content Timeline Child.
 - Once also check the single block setting display or not.
 - No changes in Copy Paste functionality, just in settings display so test accordingly.

### Checklist:
